Seedshop/Seedshop.cs

```diff
@@ -0,0 +1,363 @@
+				{ "panel", 4 },
+			}
+		},
+		{
+			"Oregano",
+			new Dictionary<string, int>
+			{
+				{ "price", 14 },
+				{ "available", 6 },
+				{ "panel", 5 },
+			}
+		},
+		{
+			"Thyme",
+			new Dictionary<string, int>
+			{
+				{ "price", 18 },
+				{ "available", 4 },
+				{ "panel", 6 },
+			}
+		},
+		{
+			"Rosemary",
+			new Dictionary<string, int>
+			{
+				{ "price", 25 },
+				{ "available", 2 },
+				{ "panel", 7 },
+			}
+		},
+		{
+			"Sage",
+			new Dictionary<string, int>
+			{
+				{ "price", 17 },
+				{ "available", 7 },
+				{ "panel", 8 },
+			}
+		},
+		{
+			"Chives",
+			new Dictionary<string, int>
+			{
+				{ "price", 13 },
+				{ "available", 10 },
+				{ "panel", 9 },
+			}
+		},
+		{
+			"Dill",
+			new Dictionary<string, int>
+			{
+				{ "price", 11 },
+				{ "available", 9 },
+				{ "panel", 10 },
+			}
+		},
+		{
+			"Lavender",
+			new Dictionary<string, int>
+			{
+				{ "price", 30 },
+				{ "available", 1 },
+				{ "panel", 11 },
+			}
+		},
+		{
+			"Tarragon",
+			new Dictionary<string, int>
+			{
+				{ "price", 22 },
+				{ "available", 4 },
+				{ "panel", 12 },
+			}
+		},
+		{
+			"Fennel",
+			new Dictionary<string, int>
+			{
+				{ "price", 16 },
+				{ "available", 6 },
+				{ "panel", 13 },
+			}
+		},
+		{
+			"Marjoram",
+			new Dictionary<string, int>
+			{
+				{ "price", 19 },
+				{ "available", 5 },
+				{ "panel", 14 },
+			}
+		},
+		{
+			"Lemon Balm",
+			new Dictionary<string, int>
+			{
+				{ "price", 14 },
+				{ "available", 8 },
+				{ "panel", 15 },
+			}
+		},
+		{
+			"Chervil",
+			new Dictionary<string, int>
+			{
+				{ "price", 12 },
+				{ "available", 10 },
+				{ "panel", 16 },
+			}
+		},
+	};
+
+	public override void _Ready()
+	{
+		this.Hide();
+		EventBus.Instance.OnSeedshopOpened += OpenSeedshop;
+		EventBus.Instance.OnSeedshopClosed += CloseSeedshop;
+		var closeButton = GetNode<Button>("CloseButton");
+		closeButton.Connect("pressed", new Callable(this, nameof(OnCloseButtonPressed)));
+
+		var GridContainer = GetNode<GridContainer>("ColorRect/GridContainer");
+		if (GridContainer != null)
+		{
+			foreach (Node ShopSlot in GridContainer.GetChildren())
+			{
+				var textureButton = ShopSlot.GetNode<TextureButton>("Panel/ColorRect/TextureButton");
+				if (textureButton != null)
+				{
+					string name = ShopSlot.Name;
+					textureButton.Connect("pressed", new Callable(this, name));
+				}
+			}
+		}
+		else
+		{
+			_logger.Error("GridContainer is null");
+		}
+
+		InitializeShopSlotLabels();
+	}
+
+	private void OpenSeedshop()
+	{
+		//work around for logging the event multiple times. On interacting, it gets called more than once
+		if (!this.Visible)
+		{
+			this.Show();
+			_logger.Debug("Seedshop opened.");
+		}
+	}
+
+	private void CloseSeedshop()
+	{
+		if (this.Visible)
+		{
+			this.Hide();
+			_logger.Debug("Seedshop closed.");
+		}
+	}
+
+	private void CustomTooltip(string name)
+	{
+		tooltip = GetNode<Label>("Seedshop/Tooltip");
```

Comment:
Didn't change anything. Needs to be discussed how exactly tooltips should look like and then if it would be a good idea to have only 1 tooltip node that gets changed accordingly or create one for each and destroy it after. 
Pro: less creating and destroying, central tooltip
Contra: maybe limited to the design and often free space if stff is unused, more constant space allocation,

Reply:
Since it's not worth the additional effort. It shall stay this way.